### PR TITLE
Leo/restart recon

### PR DESCRIPTION
**Pull request by github12364**
Reconstruction restarts! Everything should be working.

### CODE REVIEW

File: Worldepth/app/src/main/cpp/SLAM-include/System.h [hunk 1]
```diff
@@ -58,6 +58,7 @@ namespace SLAM
         // Initialize the SLAM system. It launches the Local Mapping, Loop Closing and Viewer threads.
         System(const string &strVocFile, const string &strSettingsFile);
 
+        void DeleteORBVocab();
```
Comment (ThatOneGoat) [inline Worldepth/app/src/main/cpp/SLAM-include/System.h]: Do we need this? There is no implementation in the System.cpp, so I'm assuming this is an accident

File: Worldepth/app/src/main/cpp/Reconstructor.cpp [hunk 2]
```diff
@@ -11,9 +11,9 @@ Reconstructor::Reconstructor(std::string &vocFile, std::string &settingsFile) {
 }
 
 Reconstructor::~Reconstructor() {
+    //slam should be deleted to reinit here. Need to figure out orbvocload problem first.
```
Comment (ThatOneGoat) [inline Worldepth/app/src/main/cpp/Reconstructor.cpp]: Is this comment still relevant?

Comment (github12364) [reply Worldepth/app/src/main/cpp/Reconstructor.cpp]: yes

File: Worldepth/app/src/main/cpp/SLAM-include/System.h (hunk 1 above)
Comment (github12364) [reply Worldepth/app/src/main/cpp/SLAM-include/System.h]: yeah ok will delete

**Pull request merged**